gitlab-ci/build_stage.ps1

```diff
@@ -0,0 +1,39 @@
+Param(
+    [Parameter(Mandatory = $True)]
+    [string] $pkg
+)
+
+$ErrorActionPreference="Stop"
+
+cd $PSScriptRoot/../win
+
+${ENV:SCRATCH} = "${ENV:Agent_TempDirectory}/roaster-scratch"
+mkdir "${Env:SCRATCH}"
```

Comment:
Will this work recursively? I mean is it equivalent to `mkdir -p` on Linux?

Reply:
Yes. mkdir is alias for `New-Item -Type Directory` in pwsh, and it doesn't have a -p switch.